src/cloudai/workloads/dynamo_mocker/dynamo_mocker.sh

```
@@ -0,0 +1,789 @@
+  done
+}
+
+# ── Python resolution ────────────────────────────────────────────────────
+# Called in main() after parse_args so --venv-python is available.
+# Prepends venv bin to PATH so cmd fields in the TOML (e.g.
+# "python3 -m dynamo.mocker", "aiperf profile", "genai-perf profile")
+# resolve to venv binaries without needing full paths.
+resolve_python() {
+  if [[ -n "$venv_python" && -x "$venv_python" ]]; then
+    PYTHON="$venv_python"
+    export PATH="$(dirname "$venv_python"):${PATH}"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)